payments-core/src/main/java/com/stripe/android/paymentsheet/model/SupportedPaymentMethod.kt

```diff
@@ -1,36 +1,57 @@
 package com.stripe.android.paymentsheet.model
 
+import android.os.Parcelable
 import androidx.annotation.DrawableRes
 import androidx.annotation.StringRes
 import com.stripe.android.R
 import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodCreateParams
+import com.stripe.android.paymentsheet.specifications.FormType
+import kotlinx.parcelize.Parcelize
 
 /**
  * Enum defining all payment methods supported in Payment Sheet.
  */
+@Parcelize
```

Comment:
Yes, `FormType` only exists because we need some object in `paymentsheet` to represent the form type (to avoid circular dependency). Once the Payment Sheet code is in `paymentsheet`, `FormType` won't be needed as the fragment can get the type from `SupportedPaymentMethod`.